### PR TITLE
Content Addition

### PR DESCRIPTION
**Pull request by alan2000alex**
New pages have been added to the site: Team, Gallery
Content added to the home, about, contact pages.
The theme submodule was removed.
A modified version of the theme submodule was added.

### CODE REVIEW

**Comment by realaravinth** [comment]
There are way too many grammatical inaccuracies to merge this PR. Please instruct the contributor to run their contributions through a language assistance tool(such as Grammarly) before they submit it to you.

File: content/about.md [hunk 1]
```diff
@@ -13,13 +13,32 @@ date = "2020-04-11"
 author = "Author1"
 +++ -->
 
-# Hi there
+# History
+**Null** or **n|u** is a registered not-for-profit society and an active security community with 13 chapters in major cities-
```
Comment (realaravinth) [inline content/about.md:17]: "n|u" is the logo of the main organization, I don't think it should go in here

Comment (alan2000alex) [reply content/about.md]: this has been removed

**Comment by alan2000alex** [comment]
Grammatical mistakes have been ironed out.

File: content/_index.md [hunk 2]
```diff
@@ -9,22 +9,46 @@ draft: true
...
+**October 2020**
+
+{{< figure src="/CTF1.png" caption="Capture the Flag" title="> null_" alt="> null_">}}
+Introductory event for the freshers.
+
+**November 2020**
+
+{{< figure src="/BugBounty.png" caption="Bug Bounty Webinar" title="> null_" alt="> null_">}}
+
+Mr Shashank, a security analyst in a reputed firm was here to give a talk on Building a Career in Ethical Hacking and Bug Bounties. He has amassed quite a fortune through bug bounties during his college years.
```
Comment (realaravinth) [inline content/_index.md:38]: Shashank is from Hackerone, do mention that. Also link to his blog at https://blog.shashank.co/

Comment (realaravinth) [reply content/_index.md:38]: also get rid of the random capitalization and choose a formal tone

File: content/_index.md [hunk 3]
```diff
@@ -9,22 +9,46 @@ draft: true
...
+
+**December 2020**
+
+{{< figure src="/humanrecon1.png" caption="Human Reconnaissance" title="> null_" alt="> null_">}}
+
+>New forums and communities that exist on the surface, deep and dark web have granted easy access for individuals to trade data and information often undetected until it’s too late.
+
+Human Reconnaissance is a unique service that profiles an individual’s digital footprint using open source,
```
Comment (realaravinth) [inline content/_index.md:46]: this stuff is stolen from https://cyberqgroup.com/human-recon/ where they are selling a product for human reconnaissance. 

Please form a QA team. Jarring errors such as these should be dealt with before submitting PRs.

File: content/_index.md [hunk 4]
```diff
@@ -9,22 +9,46 @@ draft: true
...
+deep web intelligence to identify online behavioural traits that could be a threat to your organization.
+
+We brought over our beloved Dr Sibi Chakkaravarthy Sethuraman (Associate Professor, VIT-AP) to help us better understand this issue.
```
Comment (realaravinth) [inline content/_index.md:49]: We can't use words like "beloved", makes us look like amateurs

Comment (alan2000alex) [reply content/_index.md]: This has been removed

File: content/_index.md (hunk 3 above)
Comment (alan2000alex) [reply content/_index.md]: All copied content has been removed.

File: content/_index.md (hunk 2 above)
Comment (alan2000alex) [reply content/_index.md]: this has been rectified

**Review by realaravinth — APPROVED** [review]

**Pull request merged**